utils/scarb-proc-macro-server-types/src/methods/mod.rs

```diff
@@ -18,16 +19,19 @@ pub trait Method {
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub struct ProcMacroResult {
```

Comment:
What would we do, if we wanted to drop support for v1? 🤔 
Why can't we pass v2 here? 🤔

Reply:
We would release another version of this crate then 
I presume i can do pass v2 and then downcast on LS side if you find that cleaner 
Although with dropping v1 i would also make `code_mappings` compulsory